NuKeeper.Update.Tests/UpdateRunnerTests.cs

```diff
@@ -0,0 +1,218 @@
+using System;
+using System.Collections.Generic;
+using NuGet.Configuration;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using NuKeeper.Abstractions.Configuration;
+using NuKeeper.Abstractions.NuGetApi;
+using NuKeeper.Abstractions.RepositoryInspection;
+using NSubstitute;
+using NuKeeper.Abstractions.Logging;
+using NuKeeper.Update.Process;
+using NUnit.Framework;
+using NuKeeper.Abstractions.NuGet;
+using System.Threading.Tasks;
+
+namespace NuKeeper.Update.Tests
+{
+    [TestFixture]
+    public class UpdateRunnerTests
+    {
+        private INuKeeperLogger _nuKeeperLogger;
+        private ISettingsContainer _settingsContainer;
+        private IFileRestoreCommand _fileRestoreCommand;
+        private INuGetUpdatePackageCommand _nuGetUpdatePackageCommand;
+        private IDotNetUpdatePackageCommand _dotNetUpdatePackageCommand;
+        private IDotNetRestoreCommand _dotNetRestoreCommand;
+        private IUpdateProjectImportsCommand _updateProjectImportsCommand;
+        private IUpdateNuspecCommand _updateNuspecCommand;
+        private IUpdateDirectoryBuildTargetsCommand _updateDirectoryBuildTargetsCommand;
+        private IUpdateRunner _sut;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _nuKeeperLogger = Substitute.For<INuKeeperLogger>();
+
+            _settingsContainer = Substitute.For<ISettingsContainer>();
```

Comment:
Just create the object here with defaults, you can then change settings in each test class